### PR TITLE
algol68g: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by maxim-belkin**

### CODE REVIEW

**Comment by maxim-belkin** [comment]
```
==> brew linkage --test algol68g
System libraries:
  /lib/x86_64-linux-gnu/libc.so.6
  /lib/x86_64-linux-gnu/libdl.so.2
  /lib/x86_64-linux-gnu/libm.so.6
  /lib/x86_64-linux-gnu/libpthread.so.0
  /usr/lib/x86_64-linux-gnu/libpq.so.5
No broken dylib links
Unwanted system libraries:
  libpq.so.5
==> FAILED
```

**Comment by sjackman** [comment]
```
Unwanted system libraries:
  libpq.so.5
```
When you see this error, use `http://packages.ubuntu.com` to identify the `lib*-dev` package that needs to be removed in the `circle.yml` and `.travis.yml` files, and then remove that package in this PR. For example
http://packages.ubuntu.com/search?searchon=contents&keywords=libpq.so&mode=&suite=trusty&arch=amd64
shows that `libpq.so` is provided by `libpq-dev`, so it should be removed from `circle.yml` and `.travis.yml`.

File: .travis.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ before_install:
       llvm-3.4
       mysql-common
       zlib1g-dev
+      libpq-dev
```
Comment (sjackman) [inline .travis.yml]: Please sort the list alphabetically.

File: circle.yml [hunk 2]
```diff
@@ -43,6 +43,7 @@ dependencies:
         llvm-3.4
         mysql-common
         zlib1g-dev
+        libpq-dev
```
Comment (sjackman) [inline circle.yml]: Ditto

File: .travis.yml (hunk 1 above)
Comment (maxim-belkin) [inline .travis.yml]: that's temporary change, no?

Comment (sjackman) [inline .travis.yml]: No, it's permanent.

File: circle.yml (hunk 2 above)
Comment (maxim-belkin) [inline circle.yml]: should I re-sort all the libs? the order would be
```
        libblas-dev
        libbz2-dev
        libfreetype6-dev
        libgdbm-dev
        libicu-dev
        libidn11-dev
        libjasper-dev
        libjbig-dev
        libkrb5-dev
        liblzma-dev
        libncurses5-dev
        libpcre3-dev
        libpq-dev
        libtinfo-dev
        libwebp-dev
        libwmf-dev
        libx11-dev
        libxml2-dev
        llvm-3.4
        mysql-common
        zlib1g-dev
```

Comment (maxim-belkin) [inline circle.yml]: disregard my question :)

**Pull request closed without merging**